.github/workflows/config/spelling_allowlist.txt

```diff
@@ -236,19 +254,21 @@ preprocessor
 probability
 programmatically
 pybind
+pyscf
 quantize
 quantized
 qubit
 qubits
 qudit
 qudits
-reStructuredText
 reconfigurable
+reStructuredText
```

Comment:
This reordering of the items in this file will cause the CI to fail. It is expecting the items to be the order that the `sort` command would output.